communityCustomizations/ShowPasswordButton/README.md

```diff
@@ -0,0 +1,18 @@
+# ShowPasswordButton 
+
+## Overview
+
+This project contains an OnLoad.js script for adding "Show Password" batton in password field on ADFS logon page.
+Works with password fields on Form based auth and with custom provider using domain password as secondary auth. 
+
+## Applying the customization
+
+To add "Show password" button do the following: 
+
+1. Add the code from Project Onload.js to your webtheme Onload.js
+
+2. Apply the customization according to https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/advanced-customization-of-ad-fs-sign-in-pages  
+
```

Comment:
Would you mind adding an "example" section with the images. You can do so like this:

## Example

![Example Screenshot](./customization1.png)

Reply:
OK